packages/cli/package.json

```diff
@@ -1,19 +1,56 @@
 {
   "name": "@stoplight/prism-cli",
   "version": "0.0.0",
-  "main": "lib/bundle.js",
-  "module": "lib/bundle.es.js",
   "author": "Stoplight <support@stoplight.io>",
+  "bin": {
+    "prism": "./bin/run"
+  },
+  "bugs": "https://github.com/chris-miaskowski/prism-cli/issues",
+  "dependencies": {
+    "@oclif/command": "^1",
```

Comment:
Can we peg deps with the `1.x.x` type syntax? And if not using that syntax elsewhere, applying the pattern there as well.

Reply:
Sure, that was automatically generated by oclif cli ;)